iphone/Classes/TiAppiOSUserActivityProxy.m

```diff
@@ -186,7 +186,7 @@ -(NSNumber*)isValid
 
 -(NSNumber*)supported
 {
-    DEPRECATED_REPLACED(@"App.IOS.UserActivity.getSupported()" ,@"5.1.0",@"App.IOS.UserActivity.isSupported()")
+    DEPRECATED_REPLACED(@"App.IOS.UserActivity.getSupported()" ,@"5.1.0",@"Titanium.App.IOS.UserActivity.isSupported()")
```

Comment:
We have a weird behavior in the macro that I discovered last week. The prefix is added to the first part, but not to the second one. So this is right for now and will be refactored along with the other used macros in the future to unify the macro.